bundle/manifests/openshift-gitops-operator-metrics-bearer-token_v1_secret.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  annotations:
+    kubernetes.io/service-account.name: openshift-gitops-operator-controller-manager
+  name: openshift-gitops-operator-metrics-bearer-token
```

Comment:
Can be name this as `openshift-gitops-operator-metrics-monitor-bearer-token` for consistency ? Sorry, I should have requested this before merging.